src/kinetics/Reaction.cpp

```diff
@@ -1245,6 +1283,28 @@ BlowersMasel readBlowersMasel(const Reaction& R, const AnyValue& rate,
     return BlowersMasel(A, b, Ta0, w);
 }
 
+ElectronTemperature readElectronTemperature(const Reaction& R, const AnyValue& rate,
+                        const Kinetics& kin, const UnitSystem& units,
+                        int pressure_dependence=0)
```

Comment:
Should be handled by a specialized `setParameters` method defined for the `ElectronTemperature` reaction rate class.